scripts/insert_fault_injection_examples.py

```diff
@@ -0,0 +1,170 @@
+from datetime import datetime
+
+from src.base_model import db
+from src.fault_injector.fault_configurations.schedule_blocked_fault_configuration import (
+    ScheduleBlockedFaultConfiguration,
+    ScheduleBlockedFaultConfigurationXSimulationConfiguration,
+)
+from src.fault_injector.fault_configurations.track_blocked_fault_configuration import (
+    TrackBlockedFaultConfiguration,
+    TrackBlockedFaultConfigurationXSimulationConfiguration,
+)
+from src.fault_injector.fault_configurations.track_speed_limit_fault_configuration import (
+    TrackSpeedLimitFaultConfiguration,
+    TrackSpeedLimitFaultConfigurationXSimulationConfiguration,
+)
+from src.fault_injector.fault_configurations.train_speed_fault_configuration import (
+    TrainSpeedFaultConfiguration,
+    TrainSpeedFaultConfigurationXSimulationConfiguration,
+)
+from src.implementor.models import SimulationConfiguration
+from src.schedule.schedule_configuration import (
+    ScheduleConfiguration,
+    ScheduleConfigurationXSimulationPlatform,
+)
+from src.spawner.spawner import (
+    SpawnerConfiguration,
+    SpawnerConfigurationXSchedule,
+    SpawnerConfigurationXSimulationConfiguration,
+)
+
+with db.atomic():
+    platforms = ["bs_0", "bs_1"]
+
+    # --- different schedules to try out the fault injection --- #
+    regular_schedule = ScheduleConfiguration.create(
+        schedule_type="TrainSchedule",
+        strategy_type="RegularScheduleStrategy",
+        strategy_start_time=0,
+        strategy_end_time=7200,
+        train_schedule_train_type="regio",
+        regular_strategy_frequency=180,
+        regular_strategy_frequency=400,
```

Comment:
You listed `regular_strategy_frequency`two times 😇

Reply:
Oh, you are right!
Oh, you are right!
🐝